apps/demo-app/src/styles/partials/main.scss

```diff
@@ -2,7 +2,7 @@
 @use '../modules/theming/theming' as app_theme;
 @use '../modules/typography/fonts' as app_font;
```

Comment:
@Cvetanovic  Is it possible to use kebab case instead of snake case when naming aliases?